src/main/java/entity/Task.java

```diff
@@ -24,7 +24,7 @@
      * @param taskName name of the task
      */
     public Task(String taskName) {
-        this(taskName, null, null);
+        this(taskName, Duration.ofHours(1), null);
```

Comment:
I think the default duration should be a constant instead of an in-method literal.